Java/src/some-package/java/com/mainpackage/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.mainpackage;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * Even though this file contains weak crypto issues, CodeGuru Reviewer will not
+ * report any issues in it, because it has been excluded in aws-codeguru-reviewer.yml.
+ * 
+ * For more information, see the Amazon CodeGuru Reviewer User Guide:
+ * https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
+ */
+public class Main {
+
+    public static void main(String[] argv) {
+        String message = argv[0];
+        String key = argv[1];
+        Cipher cipher = Cipher.getInstance("RSA");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It looks like your code uses a cipher object with an insecure transformation. To make your code more secure, use one of the following algorithms with a built-in integrity check: `AES/GCM/NoPadding` or `ChaCha20-Poly1305`.

[Learn more](https://cwe.mitre.org/data/definitions/327.html)